### PR TITLE
Tons of updates for styling the ref app to a Material design version

### PR DESCRIPTION
**Pull request by jatkinson10**
This is a big update and I recommend a thorough review.

### CODE REVIEW

File: src/main/webapp/css/main.css [hunk 1]
```diff
@@ -52,376 +212,569 @@ table {
...
+   font-size:.5em;
+   padding-top: .5em;  
+   color: #777;
 }
 
-::-webkit-scrollbar-track {
-    -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.3);
+.navigation a {
+    text-decoration: none;
+    color: #007700 !important;
```
Comment (jatkinson10) [inline src/main/webapp/css/main.css]: fixed

File: src/main/webapp/css/main.css [hunk 2]
```diff
@@ -52,376 +212,569 @@ table {
...
+    line-height: 1.1em;
+   
 }
 
-::-webkit-scrollbar-thumb {
-    -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.5);
+.navigation a:hover {
+    text-decoration: none;
+    background: #007700;
+    color: #fff !important;
```
Comment (jatkinson10) [inline src/main/webapp/css/main.css]: fixed

**Comment by codeallthethingz** [comment]
Gonna have a look at this on the plane

**Comment by codeallthethingz** [comment]
Looks amazing.  

I'd only change a few things.  
Navigation
1.  The nav values are too tall.  I'd make them much more compact.
2. The navigation boxes aren't clipped so they take up an enormous amount of real estate.  The current refapp clips them so you can see more of the different navs in one screen / don't have to scroll as much - open to other ways of making this better.  Maybe first few open, last ones closed etc...
3. The display name and the actual name of the navs collide when they're long.  

Results
1. There was a hover over function that highlighted other column's records position yellow so you can see how the things jump around.  IT might still be working, could just be that the style isn't defined.

**Comment by jatkinson10** [comment]
@willwarren please pull again as a bunch more was changed. review and let me know if I can merge.

**Comment by codeallthethingz** [comment]
@jatkinson10 can the highlight corresponding be less RED!!!!.  And more, subtle green? :)

**Comment by jatkinson10** [comment]
done @willwarren

**Comment by codeallthethingz** [comment]
:+1:

**Review by osman — APPROVED** [review]

**Pull request merged**